spec/requests/orders/complete_order_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../shared/request_not_completing_order'
+
+RSpec.describe 'PATCH admin/orders/:id?event=complete', type: :request do
+  let!(:order) { create(:order) }
+
+  context 'when logged in as administrator' do
```

Comment:
What if user login on account with different privileges?

Reply:
added authorization specs to this one, and all requests specs for `shipping `and `payment`